mono/metadata/unity-liveness.c

```diff
@@ -469,6 +694,108 @@ void mono_unity_liveness_calculation_from_statics(LivenessState *liveness_state)
 	mono_filter_objects(liveness_state);
 }
 
+void gchandle_process(void *data, void *user_data)
+{
+	MonoObject *target = data;
+	LivenessState *liveness_state = user_data;
+
+	mono_add_and_validate_object(target, liveness_state);
+}
+
+extern void
+mono_gc_strong_handle_foreach(GFunc func, gpointer user_data);
+
+static void
+foreach_thread_static_field (gpointer key, gpointer value, gpointer user_data)
+{
+	MonoClassField *field = key;
+	guint32 offset = GPOINTER_TO_UINT(value);
+	LivenessState *liveness_state = user_data;
+
+	if (!mono_field_can_contain_references(field))
+		return;
+	// TODO
```

Comment:
Do we care about this todo?

Reply:
I don't know what it's there for anymore so nope!